src/tidy.zig

```diff
@@ -59,6 +59,27 @@ test "tidy" {
     }
 }
 
+test "tidy changelog" {
+    const allocator = std.testing.allocator;
+
+    const max_bytes = 1024 * 1024;
+    const changelog = try fs.cwd().readFileAlloc(allocator, "CHANGELOG.md", max_bytes);
```

Comment:
Maybe `changelog_size_max`?

Reply:
Good call! I was mechanically applying "named parameter pattern", but of course we have more context here for the name than at the `readFileAlloc`'s definition site!